Gruntfile.js

```diff
@@ -155,15 +154,6 @@ module.exports = function (grunt) {
       }
     },
 
-    // Automatically inject Bower components into the app
-    'bower-install': {
```

Comment:
Why was this removed? The only information I have in the commit message is “Bump node to 6.9.5 and package versions” 😕

Has `bower-install` become redundant?